static_report/src/utils/layout.ts

```diff
@@ -3,7 +3,7 @@ export const breadcrumbHeight = 48;
 export const mainContentAreaHeight = `calc(100vh - ${
   topNavAndFooterHeightOffset + breadcrumbHeight
 }px)`;
-export const tableListGridTempCols = '2fr 1fr 1.5fr';
+export const tableListGridTempCols = '1fr 1fr 1.5fr';
```

Comment:
This would restrict the displayed table names width. Previously Pop showed me a dataset with super long table+column names like: 
```
SUPER_LONG_PREFIX_SUPER_LONG_NAME_SUPER_LONG_NAME_DATETIME
SUPER_LONG_PREFIX_SUPER_LONG_NAME_SUPER_LONG_NAME_DATETIME
...
```
Alot of table names can be really long, and user can lose context easily if truncated short. 
According to Pop, being able to read the table/column name is higher-priority